### PR TITLE
Add explanation of diagram for accessibility

### PR DESCRIPTION
**Pull request by cmcnallygds**
What
----

The diagram was not previously explained in full in the text on this page. This change aims to explain its content in full for users who cannot see the diagram.

How to review
-------------

Preview the content locally and check that it is accurate

Who can review
--------------

Not @schmie or @cmcnallygds

### CODE REVIEW

File: source/support/shipping_elasticsearch_metrics_to_tenants.html.md [hunk 1]
```diff
@@ -8,32 +8,45 @@ best-effort basis.
...
+* the Aiven API access credentials allow access to all services procured by GOV.UK PaaS
+* access to Aiven services is restricted to GOV.UK PaaS IP ranges
+
+To make Elasticsearch metrics provided by the Aiven Prometheus endpoints available to our tenants, GOV.UK PaaS are developing the Elasticsearch Metric Exporter app, which is currently in alpha. 
+
+The code and manifests are in a [`pull request`](https://github.com/alphagov/paas-observability-release/pull/5) on [`paas-observability-release`](https://github.com/alphagov/paas-observability-release).
+
+## How it works
+
+![](https://drive.google.com/file/d/19anv2xRk9kj4qLif06aCxbQWESHoIbVv/view)
```
Comment (AP-Hunt) [inline source/support/shipping_elasticsearch_metrics_to_tenants.html.md:31]: I think we'd be better off keeping that file in the repository

File: source/support/shipping_elasticsearch_metrics_to_tenants.html.md [hunk 2]
```diff
@@ -8,32 +8,45 @@ best-effort basis.
...
+
+The Elasticsearch Metric Exporter app runs two components, a [Prometheus server](https://prometheus.io/) and the `aiven-service-discovery` process as a [sidecar](https://docs.cloudfoundry.org/devguide/sidecars.html).
+
+The `aiven-service-discovery` process is configured by providing a JSON file that lists the services to collect metrics for. It then:
+
+* connects to the Aiven API to identify the IP addresses of the services' Elasticsearch nodes
+* configures the scrape targets of the Elasticsearch Metric Exporter Prometheus server.
+
+Once configured, the Elasticsearch Metric Exporter Prometheus server reads metrics from the Aiven Prometheus endpoints and stores them in an InnoDB instance. Access to the server is provided by a [basic auth route service](https://docs.cloud.service.gov.uk/deploying_services/route_services/#example-route-service-to-add-username-and-password-authentication).
```
Comment (AP-Hunt) [inline source/support/shipping_elasticsearch_metrics_to_tenants.html.md:40]: InnoDB is a MySQL thing. We're probably storing them in an InfluxDB service instance

File: source/support/shipping_elasticsearch_metrics_to_tenants.html.md (hunk 1 above)
Comment (schmie) [reply source/support/shipping_elasticsearch_metrics_to_tenants.html.md]: This was accidentally left in - deleted.

File: source/support/shipping_elasticsearch_metrics_to_tenants.html.md (hunk 2 above)
Comment (schmie) [reply source/support/shipping_elasticsearch_metrics_to_tenants.html.md]: I think my cerebellum took over between In and DB

**Pull request merged**